addons/purchase_stock/tests/test_reordering_rule.py

```diff
@@ -1175,3 +1175,41 @@ def test_supplierinfo_last_purchase_date(self):
         replenishment_info = self.env['stock.replenishment.info'].create({'orderpoint_id': orderpoint.id})
         supplier_info = replenishment_info.supplierinfo_ids
         self.assertEqual(supplier_info.last_purchase_date, dt.today().date(), "The last_purhchase_date should be set to the most recent date_order from the purchase orders")
+
+    def test_reordering_rule_multicurrency(self):
+        """
+            trigger a reordering rule in foreign currency
+        """
+        eur = self.env.ref('base.EUR')
+        product = self.env['product.product'].create({
+            'name': 'Super Product',
+            'type': 'product',
+        })
+        self.env['product.supplierinfo'].create({
```

Comment:
if you can reuse the product/seller from the setUpClass, you can help break the cycle of making new records in tests (i.e. speed them up)

Reply:
Ok, thanks for the advice, I updated the test avoiding as much as possible the creation of new records